sys/include/net/gnrc/ipv6/netif.h

```diff
@@ -578,6 +586,25 @@ static inline bool gnrc_ipv6_netif_addr_is_non_unicast(const ipv6_addr_t *addr)
  */
 void gnrc_ipv6_netif_init_by_dev(void);
 
+/**
+ * @brief   Sets a xtimer.
+ *
+ * @param[in] timer     The pointer to a xtimer.
+ * @param[in] offset    Offset for the timer.
+ * @param[in] msg       The pointer to the msg_t of the timer.
+ * @param[in] msg_type  The type of the message.
+ * @param[in] msg_data  The data of the message.
+ * @param[in] pid       The pid that the timer will send a message to.
+ */
+static inline void gnrc_ipv6_set_timer(xtimer_t *timer, uint32_t offset, msg_t *msg,
```

Comment:
But I'd rather would put it as a helper function in `xtimer` (because you also use it in `gnrc_sixlowpan`)